module/Motion/StraightRunner.h

```diff
@@ -26,6 +27,7 @@ class StraightRunner {
    * @param pwm PWM値
    */
   void run(double targetDistance, int pwm);
+  void runStraightToColor(int pwm, COLOR targetColor);
```

Comment:
他の関数と同様にコメントを残してほしい
あと関数名runToColorじゃだめ？

どうでもいいけど，runは目標距離，PWM値って順番で引数指定してるから，同じ流れにしたほうがいいのでは
```suggestion

  /**
   * 指定された色まで直進する関数
   * @param targetColor 目標距離
   * @param pwm PWM値
   */
  void runToColor(COLOR targetColor, int pwm);
```